hack/logsamples/viaq_journal.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@timestamp": "2023-10-17T20:46:29.048949Z",
+  "timestamp": "2023-10-17T20:46:29.048949Z",
```

Comment:
Why did @timestamp disappear for journal?

Reply:
Good catch, I can't recall off hand.    Will investigate, as I suspect I had a reason for doing so.